game/src/pregame/mod.rs

```diff
@@ -1,374 +1,46 @@
-}
-
-impl About {
-    fn new_state(ctx: &mut EventCtx, app: &App) -> Box<dyn State<App>> {
-        let col = vec![
-            ctx.style()
-                .btn_back("Home")
-                .hotkey(Key::Escape)
-                .build_widget(ctx, "back")
-                .align_left(),
-            {
-                Text::from_multiline(vec![
-                    Line("A/B STREET").display_title(),
-                    Line("Created by Dustin Carlino, Yuwen Li, & Michael Kirk"),
-                    Line("Character art by Holly Hansel"),
-                    Line(""),
-                    Line(
-                        "Data from OpenStreetMap, King County GIS, and Puget Sound Regional \
-                         Council",
-                    ),
-                    Line(""),
-                    Line(
-                        "Disclaimer: This game is based on imperfect data, heuristics concocted \
-                         under the influence of cold brew, a simplified traffic simulation model, \
-                         and a deeply flawed understanding of how much articulated buses can bend \
-                         around tight corners. Use this as a conversation starter with your city \
-                         government, not a final decision maker. Any resemblance of in-game \
-                         characters to real people is probably coincidental, unless of course you \
-                         stumble across the elusive \"Dustin Bikelino\". Have the appropriate \
-                         amount of fun.",
-                    ),
-                ])
-                .wrap_to_pct(ctx, 50)
-                .into_widget(ctx)
-                .centered_horiz()
-                .align_vert_center()
-                .bg(app.cs.panel_bg)
-                .padding(16)
-            },
-            ctx.style()
-                .btn_outline
-                .text("See full credits")
-                .build_def(ctx)
-                .centered_horiz(),
-        ];
-
-        Box::new(About {
-            panel: Panel::new_builder(Widget::custom_col(col))
-                .exact_size_percent(90, 85)
-                .build_custom(ctx),
-        })
-    }
-}
-
-impl State<App> for About {
-    fn event(&mut self, ctx: &mut EventCtx, _: &mut App) -> Transition {
-        if let Outcome::Clicked(x) = self.panel.event(ctx) {
-            match x.as_ref() {
-                "back" => {
-                    return Transition::Pop;
-                }
-                "See full credits" => {
-                    open_browser("https://github.com/a-b-street/abstreet#credits");
-                }
-                _ => unreachable!(),
-            }
-        }
-
-        Transition::Keep
-    }
-
-    fn draw_baselayer(&self) -> DrawBaselayer {
-        DrawBaselayer::Custom
-    }
-
-    fn draw(&self, g: &mut GfxCtx, app: &App) {
-        g.clear(app.cs.dialog_bg);
-        self.panel.draw(g);
-    }
-}
-
-struct Screensaver {
```

Comment:
Since the title screen now lives in `map_gui` and not `game`, we can't keep the screensaver with traffic being simulated easily. We could keep the bounce around effect, but meh. If we decide on a nicer design for the title screen and want some parts of the screensaver back, I'll restore it